src/app/modules/warranty/controllers/warranty.controller.ts

```diff
@@ -12,12 +12,19 @@ export class WarrantyController {
 	@Post()
 	async create( @Body() createWarrantyDto: CreateWarrantyDto) {
         const warranty = plainToClass(Warranty, createWarrantyDto);
+        this.emailService.sendEmail(process.env.WARRANTYEMAIL, 
+        	"Blendtec Warranty Claim Received",
+        	this.warrantyBody(warranty),
+        	[
+        	{path: warranty['serialnumber'], name: 'serialNumber'},
```

Comment:
I think this would fail linting? You're essentially bypassing the typing that typescript provides by using bracket notation. does this lint? either way change it to dot notation.